Simulating corrupted/duplicate acks

MitchellSimY opened the pull request:
Not yet completed. Still runs

**What does this PR do?**
Meant to simulate errors when sending Acks back to sender.

**Does this run?**
Currently, yes. But I still need to refine some of the errors

MitchellSimY committed:
Simulating corrupted/duplicate acks

Not yet completed. Still runs

carterclark commented on src/SenderBase.java:
```diff
@@ -104,17 +112,26 @@ protected static void Usage() {
+                System.out.println("Corrupted Ack");
                 break;
+            } else if (ackFromReceiver == previousAck) { // Duplicate Ack
+            	System.out.println("Duplicate Ack");
```
Can you add the ack number or the sequence number

carterclark commented on src/SenderBase.java:
```diff
@@ -104,17 +112,26 @@ protected static void Usage() {
+                break;
+            } else if (ackFromReceiver != startOffset && ackFromReceiver == 1){ // Corrupted Ack
+                System.out.println("received " + ackFromReceiver + " as ack, need to resend");            	
+                System.out.println("Corrupted Ack");
```
Same here for line 132

MitchellSimY committed:
Error Handling Resolving if statements

Resolved if statements in SenderBase to ensure that the RNG can route to the correct If/elseIf
